### DISCUSSION TITLE
connect to environment in another container

### DISCUSSION THREAD

**Discussion by libnine** [coder/code-server#5155] [Q&A]
I'm trying to run a Python conda environment that exists in one container in code-server which exists in another container. Does anyone have an example?

Essentially what I'm looking to do is build a conda environment with requirements and run that in one container, then build the code-server image and run that, with the code-server image binding to the conda container.

**Just to clarify**: I have a shared volume between containers. What I'd like to accomplish is running Container A's Python via Container B's code-server UI.

**Comment by jsjoeio** [comment]
Hmm...I'm not too sure how to do that. Hoping someone else will chime in!
